client/src/controllers/AutoSizeController.ts

```diff
@@ -0,0 +1,15 @@
+import { Controller } from '@hotwired/stimulus';
+import autosize from 'autosize';
+
+export class AutoSizeController extends Controller {
+  static targets = ['input'];
```

Comment:
```suggestion
```

I do not think we need the target here, as the `this.element` is sufficient to get the right DOM element.

Reply:
Ok  I have removed it.